build/root/.bazelrc

```diff
@@ -51,7 +52,6 @@ build:cross:linux_s390x --config=crosstool --platforms=@io_bazel_rules_go//go/to
 
 # --config=remote-cache enables a remote bazel cache
 # Note needs a --remote_instance_name=projects/PROJ/instances/default_instance flag
-build:remote-cache --remote_cache=remotebuildexecution.googleapis.com
```

Comment:
note to any other reviewers: this commit will need to be dropped :upside_down_face: